### DISCUSSION TITLE
Find out command belonging to a keybind

### DISCUSSION THREAD

**Discussion by remotecom** [LunarVim/LunarVim#4307] [Q&A] [answered]
How can I find out what the command is which is triggered by a specific keybind. In this case `<space>c` which closes current buffer.

I want to unbind `<space>c` and rebind that command to another keymap instead. How can I approach that?

**Comment by LostInTheLogs** [comment, marked as answer]
you can do this without knowing the command
```lua
lvim.builtin.which_key.mappings.YOUR_KEY = lvim.builtin.which_key.mappings.c
lvim.builtin.which_key.mappings.c = nil
```

you can use `:map <leader>c` to check what it's mapped to (`<Cmd>BufferKill<CR>`)
